flocker/node/agents/functional/test_cinder_behaviour.py

```diff
@@ -64,17 +77,18 @@ def test_updated_metadata_is_listed(self):
         """
         expected_metadata = {random_name(self): u"bar"}
 
-        new_volume = self.cinder_client.volumes.create(size=100,)
+        new_volume = self.cinder_volumes.create(size=100)
+        self.addCleanup(self.cinder_volumes.delete, new_volume)
```

Comment:
Is there any chance to volume will be created, but there is some error before we are notified? I guess there isn't much we can do about this.

On further consideration, I guess this isn't a regression in this issue anyway.